src/store/reducer.js

```diff
@@ -0,0 +1,118 @@
+import {
+  SEARCH,
+  FILTER_MOVIE,
+  VISIBLE,
+  LOAD_MOVIES,
+  FILTER_ID,
+  REMOVE_MOVIE,
+  NEW_MOVIE,
+  ADD_MOVIE,
+  RESET,
+  LOAD_MOVIES_BY_TITLE,
+  LOAD_MOVIES_BY_DATE,
+  LOAD_MOVIES_BY_RATING,
+} from "./actionTypes";
+
+const initialState = {
+  movies: [],
+  searchFilter: "",
+  filterMovie: "",
+  movieArray: [],
```

Comment:
you don't need this property. Ideally what you would do is:
1) Have all movies fetched on the load of the page
2) If user changes some filters or starts looking by the title, you do a request to the API with query holding data for your search
3) When data comes, you simply replace movies property in your redux state with data that came